src/pages/LiquidityMining.vue

```diff
@@ -185,18 +201,24 @@ export default defineComponent({
     });
 
     const currentWeek = computed(() => last(last(weeks)?.week.split('_')));
+    const otherNetwork = computed(() => {
+      if (networkConfig.key === '1') return 'Polygon';
```

Comment:
You could replace these with the enum `Network` from `network.ts`

`Network.MAINNET`
`Network.POLYGON`

If we need the network names, maybe we also create a map for them

Reply:
Forgot about this again